rust/inter-canister-calls/src/caller/src/lib.rs

```diff
@@ -156,8 +156,8 @@ pub async fn sign_message(message: String) -> Result<String, String> {
     // fairly low, and losing the attached cycles isn't catastrophic.
     match Call::bounded_wait(Principal::management_canister(), "sign_with_ecdsa")
         .with_arg(&request)
-        // Signing with a test key requires 10 billion cycles
-        .with_cycles(10_000_000_000)
+        // Signing with a test key requires 30 billion cycles
+        .with_cycles(30_000_000_000)
```

Comment:
The Rust CDK should be able to abstract away the actual cycles amount which is [hard-coded](https://github.com/dfinity/cdk-rs/blob/97f151057d5cee1585f95813325050b1374992c4/src/ic-cdk/src/api/management_canister/ecdsa/mod.rs#L9) in its code. But this feedback is unrelated to this PR.